<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoInput</class>
 <widget class="QWidget" name="VideoInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AGE AND GENDER DETECTOR</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40,40,40);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Bitstream Vera Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Enter Video Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="h1label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>591</width>
     <height>71</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:32pt; font-weight:600; color:#ffffff;&quot;&gt;AGE AND GENDER DETECTOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
font: 75 16pt &quot;Bitstream Vera Sans Mono&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>271</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
font: 75 16pt &quot;Bitstream Vera Sans Mono&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="video_name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>194</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Video name or Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>271</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
font: 75 16pt &quot;Bitstream Vera Sans Mono&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>MAIN MENU</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2019</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="displayFormat">
    <string>H:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Bitstream Vera Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Enter Date and Time</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>video_name</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>menuButton</tabstop>
  <tabstop>exitButton2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
